src/imu_ros2_node.cpp

```diff
@@ -50,6 +50,21 @@ void declareParameters(std::shared_ptr<rclcpp::Node>& node)
     node->declare_parameter("operation_mode", USER_PARAM_SETTING_MODE);
 }
 
+void parseArgs(int argc, char * argv[], std::string& device_name, std::string& device_trigger_name)
+{
+    if(argc < 3)
+    {
+        device_name = "adis16505";
+        device_trigger_name = "adis16505-dev0";
+    }
+    else
+    if(argc >= 3)
+    {
+        device_name = std::string(argv[1]);
+        device_trigger_name = std::string(argv[2]);
+    }
+}
+
```

Comment:
Why I like the idea to receive the device name from the user, I don't think it is a good idea to default to a specific device if the device is not given. We should improve the user experience. for example, we could scan the iio context and see if we find the requested adis device, if we cannot find it we could print a message to the user and show the list of found devices which are supported.
however, if no device is given in the argument I think it would be a good idea to scan the iio context and see if we find a supported adis device.and if we do, we can again ask the client which of the available devices should be used in ros.